airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -52,7 +52,7 @@ class KubernetesPodOperator(BaseOperator):
                                comma separated list: secret_a,secret_b
     :type image_pull_secrets: str
     :param volume_mounts: volumeMounts for launched pod
-    :type volume_mounts: list of VolumeMount
+    :type volume_mounts: list of airflow.contrib.kubernetes.volume_mount.VolumeMount
```

Comment:
```suggestion
    :type volume_mounts: list[airflow.contrib.kubernetes.volume_mount.VolumeMount]
```

Reply:
I made a correction. I also improved other similar cases in the code.